blocklib/blocks/include/gnuradio/blocks/meson.build

```diff
@@ -0,0 +1,35 @@
+blocks_headers = [
+    'api.h'
+]
+
+install_headers(blocks_headers, subdir : 'gnuradio/blocks')
+
+# TODO - export this as a subproject of gnuradio
+
+conf = configuration_data()
+conf.set('prefix', prefix)
+conf.set('exec_prefix', '${prefix}')
+conf.set('libdir', join_paths('${prefix}',get_option('libdir')))
+conf.set('includedir', join_paths('${prefix}',get_option('includedir')))
+conf.set('LIBVER', '0.0.1')
```

Comment:
Is this being used for anything? I don't see it referenced, and it looks like it's configuration data for a pkg-config file but you use Meson's generator anyway, which handles this for you.